halo2-lib-js/scripts/postTsc.js

```diff
@@ -0,0 +1,22 @@
+// Functions that are to be run after the typescript compiler runs
+
+const fs = require('fs');
+const packageJson = require('../package.json');
+const halo2WasmPackageJson = require('../../halo2-wasm/package.json');
+
+// Copies a modified version of package.json to the /dist folder
+function copyPackageJson() {
+  const packageJsonCopy = { ...packageJson };
+  delete packageJsonCopy.scripts;
+  delete packageJsonCopy.devDependencies;
+  delete packageJsonCopy.publishConfig;
+  packageJsonCopy.dependencies['@axiom-crypto/halo2-wasm'] = halo2WasmPackageJson.version;
```

Comment:
this requires that the halo2-wasm package has first been published; will that assumption always be valid?

Reply:
this might not always be valid (maybe you submit a PR with the version number bumped but it hasn't been published yet) -- hopefully this isn't an issue (you could always just toggle local in this case)?